sbom.cdx.json

```diff
@@ -0,0 +1,43 @@
+{
+  "bomFormat": "CycloneDX",
+  "specVersion": "1.6",
+  "version": 1,
+  "metadata": {
+    "authors": [
+      {
+        "name": "@VCS_SBOM_AUTHORS@"
+      }
+    ]
+  },
+  "components": [
+    {
+      "type": "library",
+      "bom-ref": "pkg:github/Mbed-TLS/mbedtls@@VCS_TAG@",
+      "cpe": "cpe:2.3:a:mbed:mbedtls:@VCS_TAG@:*:*:*:*:*:*:*",
```

Comment:
I'm not familiar with CPE and I have not validated this string, but it looks suspicious. Mbed TLS is no longer part of the “Mbed” organization, it's now part of the [TrustedFirmware](https://www.trustedfirmware.org/) organization.

Reply:
This is exactly why I wanted this file to live upstream, I missed this completely! On one hand using the `cpe:2.3:a:mbed` makes sense as it's what other CVEs have been using, but on the other hand using TrustedFirmware. According to MITRE, the CPE vendor can be `trustedfirmware` -- e.g. from https://nvd.nist.gov/products/cpe/detail/2AF395D6-6367-4EFF-A0D0-C0CB6CA99E3E

I'm happy to modify the SBOM file if you want to use the trustedfirmware CPE vendor going forward.

Reply:
> Please change to trustedfirmware

Done; thanks.

> Do you know if there's a process we should follow to get the NIST database updated?

I think the big picture idea is _this PR_; in that the upstream project says "this is what I want to use" and then the various CNAs dedupe the old ones like `cpe:2.3:a:arm:mbed_tls`. At the moment we don't have a very good way of saying "cpe:2.3:a:trustedfirmware:mbed_tls" is better than "cpe:2.3:a:arm:mbed_tls" and when https://docs.google.com/spreadsheets/d/1gKEWLxdLubOfgS1cqqY2umEX0ohYEoEy-B1i59HNVqg/edit?usp=sharing is all green we can start tidying up the CPE register. If it helps it's also something that irritates me, as I want to write a simple VEX rule for "_is this system affected_" and when there are variants of CPE you have to make the match frustratingly fuzzy.